V2 endstop relocation for stock 2.4 or Pinned AB Drives and also pinned X Y Joints for Arks MGN12 mod

hartk1213 opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
   Y endstop relocation for v2
#### How was it tested? 
 This has been tested by me and a few others on the discord for the past few months 
#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### Further notes

KiloQubit reviewed (changes requested):
Thanks for your mod!

I'm a bit confused - this is the endstop relocation mod, why are you adding step files to the pins mod?  And it looks like they're the same step files in the pins mod and this mod.

hartk1213 commented:
> Thanks for your mod!
> 
> I'm a bit confused - this is the endstop relocation mod, why are you adding step files to the pins mod? And it looks like they're the same step files in the pins mod and this mod.

the reason i added them to this pins mod is because i made a version that is for the stock A drive as well as the Pinned A drive and the step files look similar but one is for pins and one is for stock 2.4 non pinned a drive , the holes are slightly different dimensions

hartk1213 commented:
@KiloQubit i have updated this pr title to better indicate what is being added, i have added the MGN12 pinned XY joints to the Pinned mod folders, moved the endstop relocation files to the correct folder, and added links to all the mods that work together. 

let me know if there is anything that needs to be changed 

Thanks 
-hart

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md:
```diff
@@ -19,10 +18,19 @@ To complete this mod the following files will need to be reprinted
 
 **XY Joints**
+
+*NOTE: There are now 2 versions of these files , 1 set for Stock 2.4 and 1 set for  [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12)*
```
Can you make this a relative link?

    [Arkeet's MGN12 mod](../../arkeet/mgn12)

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md:
```diff
@@ -36,6 +44,11 @@ here is what the XY Joint will look like
 ![XY Joint](Images/1.png) 
 
 
+There is a new file for the A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead 
```
Change to relative link

    ... to go along with [Arkeet's MGN12 mod](../../arkeet/mgn12) that has the X endstop ...

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md:
```diff
@@ -36,6 +44,11 @@ here is what the XY Joint will look like
 
+There is a new file for the A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead 
+
+the files for the endstop relocation mod are here [Voron2.4_Y_Endstop_Relocation](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation)
```
Change to relative link

    [Voron2.4_Y_Endstop_Relocation](../Voron2.4_Y_Endstop_Relocation)

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Pins_Mod/README.md:
```diff
@@ -36,6 +44,11 @@ here is what the XY Joint will look like
+
+
 the following 5mm Pin hardware is what is needed as well 
 
```
`MGN12_xy_join_left_upper_pinnned.STL` is upside down

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation/README.md:
```diff
@@ -0,0 +1,21 @@
+This mod is for the V2 A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead and that is moving the Y endstop to the A drive this requires a 2pin jst to be soldered to the switch, 
```
Change to relative link and fix the run-on sentence.  Here's the whole paragraph rewritten:

    This mod moves the Y endstop to the A drive, to go along with [Arkeet's MGN12 mod](../../arkeet/mgn12)
    that has the X endstop on the toolhead.  This mod requires a 2 pin JST-XH to be soldered to the
    switch.  The best way I have done it is to solder the wires to the JST-XH first, install it with a dab
    of super glue, then solder the wires to the Omron switch.

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation/README.md:
```diff
@@ -0,0 +1,21 @@
+This mod is for the V2 A Drive to go along with [Arkeets MGN12](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/arkeet/mgn12) mod that has the X endstop on the toolhead and that is moving the Y endstop to the A drive this requires a 2pin jst to be soldered to the switch, 
+the best way i have done it is to solder the wires to the jst first, then install it with a dab of super glue , then solder the wires to the omron switch 
+
+There are 2 different files non pinned version
```
add a `-`

    There are 2 different files - non pinned version:

KiloQubit commented on printer_mods/hartk1213/Voron2.4_Y_Endstop_Relocation/README.md:
```diff
@@ -0,0 +1,21 @@
+There are 2 different files non pinned version
+- a_drive_frame_upper_with_jst_y_endstop.stl
+
+pinned verion since this mod also works with as well with the Pin mod here [Voron2.4_Pins_Mod](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/hartk1213/Voron2.4_Pins_Mod)
```
misspelled version, parens, relative link

    pinned version (since this mod also works with the [Voron2.4_Pins_Mod](../Voron2.4_Pins_Mod)):

KiloQubit reviewed (changes requested):
Thanks for your mod!  Comments below.

KiloQubit reviewed (approved):
Thanks for the updates!
